federated_content_connector/models.py

```diff
@@ -20,6 +20,12 @@ class CourseDetails(TimeStampedModel):
         help_text='Courserun key'
     )
 
+    course_key = models.CharField(
+        max_length=255,
+        help_text="The top top level course key associated with the courserun key",
```

Comment:
nit/typo: should this say, "The top-level course key associated with the course run key"?